### ISSUE TITLE
Sometimes clicking on the sayings menu appears blank

### ISSUE THREAD

**Issue by RussellSPG** [samtecspg/articulate#770]
### Issues
> *If you're having trouble with Articulate we definitely want to help you. If you can provide a little bit of information it will make our troubleshooting job easier.*

Having an issue (hard to reproduce because its random) where the screen will not go blank but missing the sayings until refreshed. I did get an error on the console:

![image](https://user-images.githubusercontent.com/42249324/58362581-9ac6e200-7e66-11e9-8fb4-59bb7ed3fcb6.png)

Console:
```
TypeError: e.props.onGoToUrl is not a function13.e16e8381b0ea8c4901ee.chunk.js:1:27582
```
- Did you download a release or pull from Master? Which release? 22.0
- What operating system are you running?
10 pro
- Are you running locally or in the cloud?
local
- Which browser are you using?
FF
----

**Comment by wrathagom** [comment]
Don't think I've seen these in logs before:

```
rasa_1           | 2019-05-25 00:57:37+0000 [-] "172.18.0.1" - - [25/May/2019:00:57:36 +0000] "GET / HTTP/1.1" 200 27 "-" "avast! Antivirus"
rasa_1           | 2019-05-25 00:57:40+0000 [-] "172.18.0.1" - - [25/May/2019:00:57:39 +0000] "GET /HNAP1/ HTTP/1.1" 404 232 "-" "avast! Antivirus"
```

**Comment by RussellSPG** [comment]
yes, I noticed too, but though it was normal since a lot of us would be using avast. So my AV could be occasionally blocking income requests?

**Comment by wrathagom** [comment]
I'm not quite sure what to make of it, though it could be possible.

**Comment by wrathagom** [comment]
@RussellSPG can you try to reproduce on 0.25?

**Comment by RussellSPG** [comment]
Seems to be fixed for 0.25
